sdk/digitaltwins/Azure.DigitalTwins.Core/tests/SessionRecords/ComponentTests/Component_Lifecycle.json

```diff
@@ -1,13 +1,13 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://fakeHost.api.wus2.digitaltwins.azure.net/models/dtmi%3Aexample%3Awifi%3B120276595?includeModelDefinition=true\u0026api-version=2020-05-31-preview",
+      "RequestUri": "https://https://fakeHost.api.wus2.digitaltwins.azure.net/models/dtmi%3Aexample%3Awifi%3B120276595?includeModelDefinition=true\u0026api-version=2020-05-31-preview",
```

Comment:
we might need to update the substitution logic -> `https://https://fakeHost.api.wus2.digitaltwins.azure.net`